hw1/02-palindrome.js

```diff
@@ -1,3 +1,49 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
+const resultPalindrome = document.querySelector(".resultPalindrome");
 
-elem.addEventListener('input', handleInput);
+function handleInput(event) {
+  event.preventDefault();
+  resultPalindrome.innerHTML = "";
+
+  const userInputValue = parseInt(elem.value);
+
+  if (Number.isNaN(userInputValue)) {
+    resultPalindrome.innerHTML = "";
+    return;
+  }
+
+  if (userInputValue < 0) {
+    resultPalindrome.innerHTML =
+      '<p class="text-danger">Please enter a positive number.</p>';
```

Comment:
Add the p tag using javascript and then use it throughout the code.

Reply:
Fixed